### ISSUE TITLE
TON_M and TON_Metric_* units should be denoted TONNE and TONNE_*

### ISSUE THREAD

**Issue by dr-shorthair** [qudt/qudt-public-repo#160]
"tonne" is the generally accepted name for 1000kg. It is used in SI documentation, in all non-english-speaking countries, and is recognised in most english-speaking countries. It acknowledges the fact that 'tonne' is not some deviant form of 'ton' but is defined separately in its own right. Use of TONNE would avoid all ambiguities around M or Metric suffixes. 

'Metric ton' is an alt-label at best, used only in USA and UK.

**Comment by dr-shorthair** [comment]
https://en.wikipedia.org/wiki/Tonne 
also https://www.bipm.org/utils/common/pdf/si-brochure/SI-Brochure-9.pdf - section 4 note (e)

**Comment by jhodgesatmb** [comment]
We are already working on this, and my French is not very good.

Jack Hodges, Ph.D.
Arbor Studios

**Comment by dr-shorthair** [comment]
Scroll down, it is all repeated in English ;-)

**Comment by jhodgesatmb** [comment]
That is indeed funny. When I initially got the mail and walked the link I was using my phone and the first pages, which were the only ones that said anything about English, were so small that I scrolled right by them. It wasn’t until I saw this note that I opened the link on a tablet, and of course the comments that the English version was way down.

On my way to finding the TONNE reference I noticed that the SI version of dimension vector orders the dimensions differently than in QUDT. I also noted that TONNE is a non-SI unit.

All of this is a bit moot as we decided to implement the TONNE units in parallel with the TON_Metric versions and use owl:sameAs to relate them, where appropriate. There are other naming considerations at play before we can release the updated version.

Jack
